scala3doc/resources/dotty_res/styles/diagram.css

```
@@ -1,20 +1,29 @@
 .node {
-    stroke: #333;
-    stroke-width: 2.5px;
-    fill: white;
+  stroke: #333;
+  stroke-width: 2.5px;
+  fill: white;
 }
 
 .edgeLabel {
-    fill: white;
+  fill: white;
 }
 
 .edgePath {
-    stroke: #333;
-    stroke-width: 1.5px;
-    fill: none;
+  stroke: #333;
+  stroke-width: 1.5px;
+  fill: #333;
 }
 
 #graph {
-    width: 100%;
-    height: 80%;
+  width: 100%;
+  height: 80%;
 }
+
+.diagram-class a {
+  text-decoration: underline;
+  color: #FFF;
+}
+
```

```suggestion
.diagram-class a:hover {
  color: #BFE7F3;
}
```
This change has been skipped. The rest LGTM